app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -200,35 +218,42 @@ private void recordCounter(final String metricName, final long value) {
 
     // Message handlers
 
-    private void startTicking() {
-        if (getTimers().isTimerActive(TICKER_NAME)) {
-            getSender().tell(Optional.of(new IllegalStateException("Timer already started")), getSelf());
-            return;
-        }
-        LOGGER.info().setMessage("Starting execution timer")
-            .addData("schema", _schema)
-            .addData("table", _table)
-            .addData("lookahead", _lookahead)
-            .log();
-        final Optional<Exception> executeResult = execute();
-        getTimers().startPeriodicTimer(TICKER_NAME, TICK, TICK_INTERVAL);
-        getSender().tell(executeResult, getSelf());
-    }
-
     private void tick() {
         recordCounter("tick", 1);
 
         final Instant now = _clock.instant();
         if (_schedule.nextRun(_lastRun).map(run -> run.isBefore(now)).orElse(true)) {
-            getSelf().tell(EXECUTE, getSelf());
+            final LocalDate startDate = ZonedDateTime.ofInstant(now, _clock.getZone()).toLocalDate();
```

Comment:
Do we really need a local timezone here? What happens if two boxes have different timezones? Would this work equally well just fixed to UTC?

Reply:
This is not a local time zone, it is UTC regardless of host. I initialized the clock to `Clock.systemUTC()`. I can be explicit here and just specify UTC a second time.